frontend/src/data/variables/CdcCovidProvider.ts

```diff
@@ -170,124 +166,106 @@ class CdcCovidProvider extends VariableProvider {
-    // we treat it as if it has "no data," i.e. we clear the hosp/death fields.
-    df = df
-      .generateSeries({
-        covid_deaths: (row) =>
-          row.death_unknown === row.covid_cases ? null : row.covid_deaths,
-        covid_hosp: (row) =>
-          row.hosp_unknown === row.covid_cases ? null : row.covid_hosp,
-      })
-      .resetIndex();
-
-    // Drop unused columns for simplicity.
-    df = df.dropSeries(["death_n", "death_unknown", "hosp_n", "hosp_unknown"]);
-
-    // Clear all county-level DC data. See issue for more details:
-    // https://github.com/SatcherInstitute/health-equity-tracker/issues/872.
-    // TODO - fix this the right way.
-    df = df.withSeries({
-      covid_cases: (df) =>
-        df.deflate((row) =>
-          row.fips === DC_COUNTY_FIPS ? null : row.covid_cases
-        ),
-      covid_deaths: (df) =>
-        df.deflate((row) =>
-          row.fips === DC_COUNTY_FIPS ? null : row.covid_deaths
-        ),
-      covid_hosp: (df) =>
-        df.deflate((row) =>
-          row.fips === DC_COUNTY_FIPS ? null : row.covid_hosp
-        ),
-    });
-
-    df = df
-      .generateSeries({
-        // Calculate per100k
-        covid_cases_per_100k: (row) =>
-          this.calculations.per100k(row.covid_cases, row.population),
-        covid_deaths_per_100k: (row) =>
-          this.calculations.per100k(row.covid_deaths, row.population),
-        covid_hosp_per_100k: (row) =>
-          this.calculations.per100k(row.covid_hosp, row.population),
-        //  Correct types for Age-Adjusted Ratios
-        hosp_ratio_age_adjusted: (row) =>
-          row.hosp_ratio_age_adjusted == null
-            ? null
-            : row.hosp_ratio_age_adjusted,
-        death_ratio_age_adjusted: (row) =>
-          row.death_ratio_age_adjusted == null
-            ? null
-            : row.death_ratio_age_adjusted,
-      })
-      .resetIndex();
-
-    ["covid_cases", "covid_deaths", "covid_hosp"].forEach((col) => {
-      df = this.calculations.calculatePctShare(
-        df,
-        col,
-        col + "_share",
-        breakdownColumnName,
-        ["fips"]
+      // If a given geo x breakdown has all unknown hospitalizations or deaths,
+      // we treat it as if it has "no data," i.e. we clear the hosp/death fields.
+      df = df
+        .generateSeries({
+          covid_deaths: (row) =>
+            row.death_unknown === row.covid_cases ? null : row.covid_deaths,
+          covid_hosp: (row) =>
+            row.hosp_unknown === row.covid_cases ? null : row.covid_hosp,
+        })
+        .resetIndex();
+
+      // Drop unused columns for simplicity.
+      df = df.dropSeries([
+        "death_n",
+        "death_unknown",
+        "hosp_n",
+        "hosp_unknown",
+      ]);
+
+      df = df
+        .generateSeries({
+          // Calculate per100k
+          covid_cases_per_100k: (row) =>
+            this.calculations.per100k(row.covid_cases, row.population),
+          covid_deaths_per_100k: (row) =>
```

Comment:
all this is only still here for national level frontend calculations, right?

Reply:
yup! Will delete it in the next story